pkg/eve/networks.go

```diff
@@ -17,7 +19,7 @@ import (
 type NetInstState struct {
 	Name        string
 	UUID        string
-	NetworkType config.ZNetworkInstType
+	NetworkType string
```

Comment:
Why would we want to switch to string?

Reply:
In my experiments I received value of variable here (some number), not a meaningful text (ZnetInstLoca in my sample). It is quite tricky to format it on-fly in marshaller